storage/tail/28-tail.html

```diff
@@ -41,10 +47,10 @@ <h3>Outputs</h3>
             name: {value:""},
             filetype: {value:"text"},
             split: {value:"[\\r]{0,1}\\n"},
-            filename: {value:"",required:true}
+            filename: {value:""}
         },
         color:"BurlyWood",
-        inputs:0,
+        inputs:1,
```

Comment:
I think it may be possible to make this dynamic based on filename field being empty or not

Reply:
I tried a bunch of stuff to get this working. Didn't work in the end.
Even tried implementing parts from here:
https://github.com/node-red/node-red-nodes/blob/master/social/email/61-email.html#L326

Reply:
Yes the email node should be an example that works.

Reply:
Fixed with 82edc64.
Input are now dynamic changed according to filename defined or not.